src/streamlink_cli/output.py

```diff
@@ -91,6 +91,9 @@ def __init__(self, cmd, args=DEFAULT_PLAYER_ARGUMENTS, filename=None, quiet=True
         self.namedpipe = namedpipe
         self.http = http
         self.title = title
+        if self.title is not None:
+            # straiten up the encoding
+            self.title = maybe_decode(self.title)
```

Comment:
```
C:\Python27\Scripts\streamlink.exe -l debug twitch.tv/twitch worst --title "ä"
[cli][debug] OS:         Windows 10
[cli][debug] Python:     2.7.16
[cli][debug] Streamlink: 2444-fix-unicode-title
...
[stream.hls][debug] Adding segment 6897 to queue
[stream.hls][debug] Download of segment 6895 complete
[cli][info] Starting player: "C:\Program Files\VideoLAN\VLC\vlc.exe"
[cli][info] Closing currently open stream...
[stream.segmented][debug] Closing worker thread
[stream.segmented][debug] Closing writer thread
Traceback (most recent call last):
  File "C:\Python27\Scripts\streamlink-script.py", line 11, in <module>
    load_entry_point('streamlink', 'console_scripts', 'streamlink')()
  File "c:\streamlink-2444-fix-unicode-title\src\streamlink_cli\main.py", line 1024, in main
    handle_url()
  File "c:\streamlink-2444-fix-unicode-title\src\streamlink_cli\main.py", line 602, in handle_url
    handle_stream(plugin, streams, stream_name)
  File "c:\streamlink-2444-fix-unicode-title\src\streamlink_cli\main.py", line 455, in handle_stream
    success = output_stream(plugin, stream)
  File "c:\streamlink-2444-fix-unicode-title\src\streamlink_cli\main.py", line 316, in output_stream
    output = create_output(plugin)
  File "c:\streamlink-2444-fix-unicode-title\src\streamlink_cli\main.py", line 125, in create_output
    record=record, title=title)
  File "c:\streamlink-2444-fix-unicode-title\src\streamlink_cli\output.py", line 96, in __init__
    self.title = maybe_decode(self.title)
  File "c:\streamlink-2444-fix-unicode-title\src\streamlink\utils\encoding.py", line 28, in maybe_decode
    return text.decode(encoding)
  File "c:\python27\lib\encodings\utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xe4 in position 0: unexpected end of data
```

Reply:
Thanks. I think I will need to set up a Windows VM to debug this :)